Source/WebCore/rendering/style/StyleRareInheritedData.cpp

```diff
@@ -23,6 +23,7 @@
 #include "StyleRareInheritedData.h"
 
 #include "CursorList.h"
+#include "ListStyleType.h"
```

Comment:
This is already included in StyleRareInheritedData.h, does it need another include here?